<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="1">

  <testsuite name="createModel_test"
    errors="0" failures="1" hostname="" id="" package="createModel_test" skipped="0"
    tests="1" time="16.47" timestamp="Thu, 14 May 2015 00:35:04 GMT">
  
    <testcase name="Create Model test" time="16.47" assertions="28">

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

      <failure message="Testing if element &lt;button[data-hook=&#39;remove-field]&gt; is present.">Testing if element &lt;button[data-hook=&#39;remove-field]&gt; is present.: Expected &quot;present&quot; but got: &quot;null&quot;
    at Object.&lt;anonymous&gt; (C:\Users\Michael\Desktop\nightwatch-testing\nightwatch\lib\core\assertion.js:105:19)
    at HttpRequest.&lt;anonymous&gt; (C:\Users\Michael\Desktop\nightwatch-testing\nightwatch\lib\index.js:299:20)
    at HttpRequest.emit (events.js:110:17)
    at HttpRequest.&lt;anonymous&gt; (C:\Users\Michael\Desktop\nightwatch-testing\nightwatch\lib\index.js:346:15)
    at HttpRequest.emit (events.js:118:17)
    at IncomingMessage.&lt;anonymous&gt; (C:\Users\Michael\Desktop\nightwatch-testing\nightwatch\lib\http\request.js:150:16)
    at IncomingMessage.emit (events.js:129:20)
    at _stream_readable.js:908:16
    at process._tickCallback (node.js:355:11)</failure>

    

    

    

    

    

    

    

    

    

    

    
    </testcase>
  
  
  </testsuite>
</testsuites>
